package.json

```diff
@@ -60,9 +60,9 @@
     "typescript": "4.3.5"
   },
   "scripts": {
-    "check-lint": "eslint --format=pretty . && prettier --check .",
+    "check-lint": "eslint --format=pretty --ext=.js,.mjs,.cjs . && prettier --check .",
     "load-fixture-repo": "node fixtures/load-repo",
-    "lint": "eslint --format=pretty --fix . && prettier --write .",
+    "lint": "eslint --format=pretty --ext=.js,.mjs,.cjs --fix . && prettier --write .",
```

Comment:
This should be the default but ESLint has been really really slow to adopt this:

https://github.com/eslint/eslint/issues/9592
https://github.com/eslint/eslint/issues/13301
https://github.com/eslint/eslint/issues/14588